packages/gestalt/src/icons/file-unknown.svg

```diff
@@ -0,0 +1 @@
+<svg height="24" width="24" xmlns="http://www.w3.org/2000/svg"><g fill="#8e8e8e"><path d="M18.5 20.5h-13v-17H14v3A1.5 1.5 0 0 0 15.5 8h3zm1.914-14.086l-4.828-4.828A2 2 0 0 0 14.172 1H5.5A2.5 2.5 0 0 0 3 3.5v17A2.5 2.5 0 0 0 5.5 23h13a2.5 2.5 0 0 0 2.5-2.5V7.828a2 2 0 0 0-.586-1.414z"/><path d="M11.754 8.37C10.235 8.37 9 9.578 9 11.06a.754.754 0 1 0 1.508 0c0-.65.56-1.18 1.246-1.18S13 10.41 13 11.06c0 .397-.23.802-.559.985-.835.465-1.44 1.55-1.44 2.58a.754.754 0 1 0 1.507 0c0-.524.351-1.086.665-1.261.811-.45 1.335-1.355 1.335-2.304 0-1.482-1.235-2.688-2.754-2.688M11.754 16.25a.875.875 0 1 0 .001 1.75.875.875 0 0 0 0-1.75"/></g></svg>
```

Comment:
This one doesnt look right, it has 2 `path` elements, are these from PDS?

Reply:
Yes. I download from it by myself and compared and it just has 2 paths.

Reply:
I have contacted Vickie and update this icon! :)